core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -82,6 +82,16 @@ public ColumnVisibility convert(String value) {
     }
   }
 
+  /**
+   * A catch all for older legacy options that have been dropped.  Most of them were replaced with
+   * accumulo-client.properties in 2.0.  Others have been dropped completely.
+   */
+  @Parameter(names = {"-p", "-tc", "--tokenClass", "-i", "--instance", "--debug", "-fake",
```

Comment:
Not 100% sure about this, but I think some of these used to be boolean and therefore the parameter did not take an argument.  If this is true, it may make sense to have two groups one with with a boolean and one with a String.

Reply:
Ah yeah thats a good idea.  I will experiment with the code and see what is easiest.